@codexteam/ui/src/vue/components/chart/Chart.vue

```diff
@@ -0,0 +1,663 @@
+  }
+}
+
+/**
+ * Formats number with spaces
+ *
+ * @param value - number value
+ */
+function formatSpacedNumber(value: number): string {
+  return new Intl.NumberFormat('ru-RU').format(value);
+}
+
+onMounted(() => {
+  /**
+   * Cache wrapper width
+   */
+  computeWrapperSize();
+
+  window.addEventListener('resize', onResize);
+});
+
+onBeforeUnmount(() => {
+  window.removeEventListener('resize', onResize);
+});
+</script>
+
+<style>
+  .chart {
+    position: relative;
+    z-index: 0;
+    display: flex;
+    flex-direction: column;
+    height: 215px;
+    background-color: var(--base--bg-secondary);
+    border-radius: 3px;
+
+    --legend-height: 40px;
+    --legend-line-height: 11px;
+    --legend-block-padding: 15px;
+
+    &__info {
+      position: absolute;
+      top: 15px;
+      right: 15px;
+      padding: 5px 10px;
+      color: var(--base--text);
+      font-size: 13px;
+      white-space: nowrap;
+      background: color-mod(var(--base--bg) alpha(50%));
+      border-radius: 5px;
+
+      &-today {
+        color: var(--base--text-secondary);
+      }
+
+      &-highlight {
+        margin-left: 6px;
+        font-weight: bold;
+      }
+    }
+
+    &__body {
+      flex-grow: 2;
+    }
+
+    &__ox {
+      height: var(--legend-height);
+      padding-block: var(--legend-block-padding);
+
+      &-inner {
+        position: relative;
+        width: 100%;
+        height: 100%;
+      }
+
+      &-item {
+        position: absolute;
+        left: 0;
+        color: var(--base--text);
+        font-size: 11px;
+        line-height: var(--legend-line-height);
+        text-align: center;
+        transform-origin: center;
+        opacity: 0.3;
+
+        &:first-of-type,
+        &:last-of-type {
+          display: none;
+        }
+      }
+    }
+
+    &__pointer {
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 0;
+      width: 3px;
+      height: 100%;
+      margin-left: -1.5px;
+      background-color: rgba(25, 28, 37, 0.5);
+      animation: pointer-in 200ms ease;
+      will-change: opacity, transform;
+
+      &-cursor {
+        position: absolute;
+        top: 0;
+        width: 7px;
+        height: 7px;
+        margin-top: -3.5px;
+        margin-left: -2px;
+        border-radius: 50%;
+        opacity: 1;
+        will-change: transform;
+      }
+
+      &-tooltip {
+        --tooltip-block-padding: 6px;
+
+        position: absolute;
+        top: calc(100% - var(--legend-height) + var(--legend-block-padding) - var(--tooltip-block-padding) - 1px);
+        left: 50%;
+        z-index: 500;
+        padding-block: var(--tooltip-block-padding);
+        padding-inline: 8px;
+        color: var(--base--text);
+        font-size: 12px;
+        line-height: 1.4;
+        letter-spacing: 0.2px;
+        white-space: nowrap;
+        text-align: center;
+        background: #191C25;
+        border-radius: 7px;
+        box-shadow: 0 7px 12px 0 rgba(0, 0, 0, 0.12);
```

Comment:
Use colors only from the design system to it adjustable for an any theme

<img width="1030" height="736" alt="Image" src="https://github.com/user-attachments/assets/4ebab22e-c2e6-4a39-a6db-6d0d81836e4d" />